src/mplfinance/_utils.py

```diff
@@ -551,13 +618,12 @@ def _construct_pointnfig_collections(dates, highs, lows, volumes, config_pointnf
         x = [index] * (diff)
         y = [curr_price + (i * box_size * sign) for i in range(start_iteration, diff+start_iteration)]
         
-        
         curr_price += (box_size * sign * (diff))
         box_values.append(sum(y) / len(y))
         
         for i in range(len(x)): # x and y have the same length
             height = box_size * 0.85
-            width = (50/box_size)/len(new_dates)
+            width = 0.6
```

Comment:
0.6 works because the plots already scale with the density of the data so we don't need to take the density into account for the width.  I tested all the data in the examples/data directory and some of my own additional data to come to this conclusion.